recipes/cobble/LICENSE

```diff
@@ -0,0 +1,12 @@
+BSD-2-Clause
```

Comment:
`cobble` doesn't currently commit the LICENSE file. They however reference it in the README of that repo [here](https://github.com/mwilliamson/python-cobble/tree/a4f92a3?tab=readme-ov-file#license).